rllib/policy/torch_policy_v2.py

```diff
@@ -859,13 +876,19 @@ def get_tower_stats(self, stats_name: str) -> List[TensorStructType]:
             of the tower's `tower_stats` dicts.
         """
         data = []
-        for tower in self.model_gpu_towers:
-            if stats_name in tower.tower_stats:
+        for model in self.model_gpu_towers:
+            if self.tower_stats:
```

Comment:
can you add some comments about these 2 cases. most importantly, which cases they are trying to cover like ModelV2 vs RLModule.